mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -327,6 +336,15 @@ impl MmCtx {
         Ok(())
     }
 
+    #[cfg(not(target_arch = "wasm32"))]
+    pub async fn init_async_sqlite_connection(&self) -> Result<(), String> {
+        let sqlite_file_path = self.dbdir().join("KOMODEFI.db");
+        log::debug!("Trying to open SQLite database file {}", sqlite_file_path.display());
```

Comment:
This may confuse people as it only prints relative paths.

For absolute path, try:

```suggestion
        log::debug!(
            "Trying to open SQLite database file {}",
            try_s!(sqlite_file_path.canonicalize()).display()
        );
```

Reply:
Done